pkg/kubectl/cmd/run.go

```diff
@@ -220,18 +220,13 @@ func Run(f cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer, cmd *cobr
 		case api.RestartPolicyOnFailure:
 			if contains(resourcesList, batchv1.SchemeGroupVersion.WithResource("jobs")) {
 				generatorName = "job/v1"
-			} else if contains(resourcesList, v1beta1.SchemeGroupVersion.WithResource("jobs")) {
```

Comment:
Correct, that's a followup to this one. I need to check the entire doc and make sure it's not using extensions/v1beta anywhere.